.github/workflows/forge-stable.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
       FORGE_NAMESPACE: forge-three-region
       FORGE_RUNNER_DURATION_SECS: 1800
       FORGE_TEST_SUITE: three_region_simulation
-      POST_TO_SLACK: true
+      POST_TO_SLACK: ${{ needs.determine-test-metadata.outputs.BRANCH == 'main' && true || false }} # only post to slack on main branch
```

Comment:
why `&& true` though? Is it redundant?

Reply:
actually yea I think you can use the bool directly. I originally thought GHA didn't allow bool in this context, only string